modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue

```diff
@@ -171,14 +174,11 @@ export default {
       const pagesSet = klona(this.pages);
 
       pagesSet.forEach(page => {
-        const data = {};
-
-        this.headers.forEach(column => {
-          data[column.property] = page[column.property];
-          data._id = page._id;
-          data.children = page.children;
-          data.parked = page.parked;
-        });
+        const data = {
+          // Clone but don't omit anything, we use a lot of properties of the page
```

Comment:
We already have klona in this component. I'd suggest using that if we're simply cloning.